res/database/Default/database_scripts.xml

```diff
@@ -470,76 +470,582 @@
+							<en>a social hotspot|an interesting meeting place|a meeting place for a wide variety of people|a place where worlds collide|a place where all social classes clash|a place where you meet all kinds of different people|a place where you meet the most different opinions|a place where you meet people from everywhere</en>
+						</asocialhotspot>
+						<discuss>
+							<de>diskutieren|debattieren|streiten|streiten sich|fetzen sich|zanken sich</de>
+							<en>discuss|debate|argue about|fight about|quarrel about|bicker about</en>
+						</discuss>
+						<fiercely>
+							<de>heftig|scharf|grimmig|wild|leidenschaftlich|böse|erbittert|hitzig|angespannt|angeregt|aufgeregt</de>
+							<en>fiercely|harshly|ferociously|wildly|passionately|madly|grimly|fierily|tensely|agitatedly|excitedly</en>
+						</fiercely>
+						<politics>
+							<de>Innenpolitik|Außenpolitik|Religion|den kategorischen Imperativ|den Klimawandel|Migranten|Flüchtlingspolitik|die wahre Form der Erde</de>
+							<en>domestic politics|foreign affairs|religion|the categorical imperative|climate change|migrants|refugee policy|the true shape of the Earth</en>
+						</politics>
 					</variables>
 				</scripttemplate>
 				<scripttemplate index="2" product="2" licence_type="2" guid="scripttemplate-random-ron-averagedayseries01_03">
 					<title>
-						<de>%SUBJECT2A% im %SUBJECT1B%</de>
-						<en>%SUBJECT2A% in %SUBJECT1B%</en>
+						<de>%meeting% im ${location_${k}}</de>
+						<en>%meeting% in ${location_${k}}</en> <!-- TODO: apply capitalization function when available -->
 					</title>
+					<description>
+						<de>%descriptions%</de>
+					</description>
 					<episodes min="1" max="3" />
 					<variables>
-						<subject2a>
-							<de>Treffen|Krawall|Klamauk|Überraschungsfeier</de>
-							<en>Meeting|Riot|Fuss|Surprise party</en>
-						</subject2a>
-						<subject1b>
-							<de>Paradies|Gasthaus|Hinterhof|Polizeirevier|Supermarkt</de>
-							<en>paradise|the roadhouse|the backyard|the police station|the supermarket</en>
-						</subject1b>
+						<descriptions>
+							<de>%description_0%|%description_1%</de>
+						</descriptions>
+						<description_0>
+							<de>Im ${location_${k}}%allstillright% ${whythere_${k}}</de>
+							<en>In ${location_${k}}%allstillright% ${whythere_${k}}</en>
+						</description_0>
+						<description_1>
+							<de>%bar% ist in %town% gerade %thepartylocation%. %musicsession%</de>
+							<en>In %town%, %bar% currently is %thepartylocation%. %musicsession%</en>
+						</description_1>
+						<meeting>
+							<de>Treffen|Aufruhr|Klamauk|Überraschungsfeier|Versammlung|Ringelpiez|Trubel|Wiedersehen|Begegnung|Spektakel|Scherz|Ereignis|Belustigung</de>
+							<en>Meeting|Riot|Fuss|Surprise party|Gathering|Revelry|Hustle and bustle|Reunion|Encounter|Spectacle|Frolic|Happening|Merrymaking</en>
+						</meeting>
+						<j>
+							<de>0|1|2|3|4|5|6|7|8|9|a|b|c|d|e|f|g|h</de> <!-- must be same as i -->
```

Comment:
"i" defined?

Reply:
yes, it's defined above, globally for the series